src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -29,13 +29,6 @@
 using Newtonsoft.Json.Linq;
 using ProtoCore.Namespace;
 
-// Internals are visible to the Package Manager extension
-// For workspace package dependency collection
-[assembly: InternalsVisibleTo("DynamoPackages")]
```

Comment:
I moved these to assemblyInfo.cs so they are easier to find all in one place. This still applies to the entire assembly even if they are only located in one file.